rlclientlib/azure_factories.cc

```diff
@@ -10,113 +10,135 @@
-    *retval = new http_transport_client<apim_http_authorization>(
-      client,
-      tasks_limit,
-      max_http_retries,
-      trace_logger,
-      error_cb);
-    return error_code::success;
-  }
-
-  // Creates i_sender object for sending observations data to the apim endpoint.
-  int observation_api_sender_create(i_sender** retval, const u::configuration& cfg, error_callback_fn* error_cb, i_trace* trace_logger, api_status* status) {
-    const auto api_host = cfg.get(name::OBSERVATION_HTTP_API_HOST, "localhost:8080");
-    return create_apim_http_api_sender(retval, cfg, api_host, cfg.get_int(name::OBSERVATION_APIM_TASKS_LIMIT, 16), cfg.get_int(name::OBSERVATION_APIM_MAX_HTTP_RETRIES, 4), error_cb, trace_logger, status);
-  }
-
-  // Creates i_sender object for sending interactions data to the apim endpoint.
-  int interaction_api_sender_create(i_sender** retval, const u::configuration& cfg, error_callback_fn* error_cb, i_trace* trace_logger, api_status* status) {
-    const auto api_host = cfg.get(name::INTERACTION_HTTP_API_HOST, "localhost:8080");
-    return create_apim_http_api_sender(retval, cfg, api_host, cfg.get_int(name::INTERACTION_APIM_TASKS_LIMIT, 16), cfg.get_int(name::INTERACTION_APIM_MAX_HTTP_RETRIES, 4), error_cb, trace_logger, status);
-  }
-
-  // Creates i_sender object for sending observations data to the event hub.
-  int observation_sender_create(i_sender** retval, const u::configuration& cfg, error_callback_fn* error_cb, i_trace* trace_logger, api_status* status) {
-    const auto eh_host = cfg.get(name::OBSERVATION_EH_HOST, "localhost:8080");
-    const auto eh_name = cfg.get(name::OBSERVATION_EH_NAME, "observation");
-    const auto eh_url = build_eh_url(eh_host, eh_name);
-    i_http_client* client;
-    RETURN_IF_FAIL(create_http_client(eh_url.c_str(), cfg, &client, status));
-    *retval = new http_transport_client<eventhub_http_authorization>(
-      client,
-      cfg.get_int(name::OBSERVATION_EH_TASKS_LIMIT, 16),
-      cfg.get_int(name::OBSERVATION_EH_MAX_HTTP_RETRIES, 4),
-      trace_logger,
-      error_cb);
-    return error_code::success;
-  }
-
-  // Creates i_sender object for sending interactions data to the event hub.
-  int interaction_sender_create(i_sender** retval, const u::configuration& cfg, error_callback_fn* error_cb, i_trace* trace_logger, api_status* status) {
-    const auto eh_host = cfg.get(name::INTERACTION_EH_HOST, "localhost:8080");
-    const auto eh_name = cfg.get(name::INTERACTION_EH_NAME, "interaction");
-    const auto eh_url = build_eh_url(eh_host, eh_name);
-    i_http_client* client;
-    RETURN_IF_FAIL(create_http_client(eh_url.c_str(), cfg, &client, status));
-    *retval = new http_transport_client<eventhub_http_authorization>(
-      client,
-      cfg.get_int(name::INTERACTION_EH_TASKS_LIMIT, 16),
-      cfg.get_int(name::INTERACTION_EH_MAX_HTTP_RETRIES, 4),
-      trace_logger,
-      error_cb);
-    return error_code::success;
-  }
+	namespace m = model_management;
```

Comment:
there are tab/white space issues in this file and the others (4indencts vs 8 indents) change causing the diff. We should revert it to see the real changes.

Reply:
fixed